opentelemetry-sdk/src/opentelemetry/sdk/trace/sampling.py

```diff
@@ -404,7 +404,7 @@ def __init__(self, rate: float):
 }
 
 
-def _get_from_env_or_default() -> Sampler:
+def get_known_sampler_from_env_or_default() -> Sampler:
```

Comment:
Why is this name made public?

Reply:
I think it was just an oversight, changing this back.